src/lib/trust_system/peer_trust.ml

```diff
@@ -126,7 +126,7 @@ struct
           Logger.faulty_peer_without_punishment logger ~module_:__MODULE__
             ~location:__LOC__
             ~metadata:
-              ( [ ("peer", Peer_id.to_yojson peer)
```

Comment:
You need to also update the interpolation of `$peer` in the message string below.